mava/systems/tf/maddpg_scaled/execution.py

```diff
@@ -0,0 +1,184 @@
+# python3
```

Comment:
Once the code is approved, it will become the default MADDPG right? Or would there be `maddpg` and `maddpg_scaled`?

Reply:
I changed maddpg_scaled to MADDPG :+1: There is now just one MADDPG algorithm that can scaled or run as normal with one trainer.